.github/workflows/build.yml

```diff
@@ -118,3 +118,20 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           RUNTIME_PARAM: ${{ matrix.runtime-param }}
         run: ./scripts/run_integration_tests.sh
+
+      - name: Send success metric
+        env:
+          DD_API_KEY_SERVERLESS: ${{ secrets.DD_API_KEY_SERVERLESS }}
```

Comment:
a new secret was added not to overwrite the existing DD_API_KEY secret but maybe we should change the existing one to point to the serverless Org and use only one DD_API_KEY?

Reply:
Do you know the whole DD_API_KEY? I think we could look it up in postgres

Reply:
fyi, we are able to confirm that this linked to an incorrect org, I'll replace the secret and use just one

Reply:
Yeah, I would have guessed that the API key in here would already be for the Serverless org 🤔 I definitely don't think we need two different API keys in the tests.